gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -79,6 +79,159 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    var floydWarshallTestsLength = 18;
+    var floydWarshallTests = [
+        {
+            name: "Initial 4 vertice, 3 edges",
```

Comment:
I've updated it, although, just to note that I'm not completely crazy, vertices does seem to be an accepted plural for vertex (although I was still missing the s on the end).

https://en.wikipedia.org/wiki/Vertex_(geometry)